server/internal/usecase/interactor/common.go

```diff
@@ -51,7 +51,7 @@ func NewContainer(r *repo.Container, g *gateway.Container,
 		Tag:          NewTag(r),
 		StoryTelling: NewStorytelling(r, g),
 		Workspace:    accountinteractor.NewWorkspace(ar, workspaceMemberCountEnforcer(r)),
-		User:         accountinteractor.NewUser(ar, ag, config.SignupSecret, config.AuthSrvUIDomain),
+		User:         accountinteractor.NewMultiUser(ar, ag, config.SignupSecret, config.AuthSrvUIDomain, ar.Users),
```

Comment:
Please update the corresponding test in e2e https://github.com/reearth/reearth/blob/040ad9055db2fe7cd10037428bd1142be2941774/server/e2e/common.go#L42